### ISSUE TITLE
PDF in HTML5 is displaying small squares instead of spaces

### ISSUE THREAD

**Issue by GoogleCodeExporter** [fleszar1/flexpaper#324]
```
What steps will reproduce the problem?

Please access http://pesquisadev.dotlib.com.br/leitura.html

1.PDF in HTML5 is displaying small squares instead of spaces, when accessed 
from the following browsers:

Windows 7 OS
- Chrome 22.0.1.229.94 m
- Firefox 16.0.1

Mac OSX
- Firefox 16.01

Android 4.0
- Firefox

2. Another PDF is displayed instead of mine, when accessed from  Opera 12.02


What is the expected output? What do you see instead?
The correct PDF being displayed on all browsers

What version of the product are you using? On what operating system?
2.0.0 (2.0.2)
Windows 7 and Mac OSX

Please provide any additional information below.
My website: http://pesquisadev.dotlib.com.br/leitura.html

Thanks,
Marcelo

```

Original issue reported on code.google.com by `marcelo....@dotlib.com` on 24 Oct 2012 at 6:52

**Comment by GoogleCodeExporter** [comment]
```
Thanks for reporting this - just double checking, are you using the latest 
FlexPaper 2.0.2 build?
```

Original comment by `erik.eng...@devaldi.com` on 24 Oct 2012 at 6:55

**Comment by GoogleCodeExporter** [comment]
```
Yes we are.
```

Original comment by `marcelo....@dotlib.com` on 24 Oct 2012 at 6:56

**Comment by GoogleCodeExporter** [comment]
```
Ok thanks - will have a look at this. Please be aware that the HTML5 rendering 
mode is in late beta and can have issues like with this example, for more 
stable viewing we recommend using HTML4 if possible
```

Original comment by `erik.eng...@devaldi.com` on 24 Oct 2012 at 6:58

**Comment by GoogleCodeExporter** [comment]
Original comment by `erik.eng...@devaldi.com` on 24 Oct 2012 at 6:58
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
Hello Marcelo,

have you been able to solve the issue in the meantime?
I see the document at http://pesquisadev.dotlib.com.br/leitura.html is rendered 
correctly now?

We have a similar issue and assuming the incorrect handling for non-Ansi 
encoding of the font in the PDF document, we are using.

If you had a solution, could you, please, share it with us?

Thanks in advance!

Kind regards
Sergiy


Sergiy Shyrkov
Product Development
---------------------------------
Jahia Solutions Group
web: http://www.jahia.com
---------------------------------
Jahia's next-generation, open source CMS stems from a widely acknowledged 
vision of enterprise application convergence – web, search, document, social 
and portal – unified by the simplicity of web content management. 
```

Original comment by `sshyr...@jahia.com` on 29 Oct 2012 at 11:05

**Comment by GoogleCodeExporter** [comment]
```
Fixed in upcoming build 2.03 scheduled for release early next week. Contact us 
if you would like a pre-build
```

Original comment by `erik.eng...@devaldi.com` on 30 Oct 2012 at 11:59
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
Hi Sergiy,

We have not.
We replaced the PDF that was having trouble with that one.
```

Original comment by `marcelo....@dotlib.com` on 31 Oct 2012 at 10:06

**Comment by GoogleCodeExporter** [comment]
```
Great, thanks a lot.

Marcelo
```

Original comment by `marcelo....@dotlib.com` on 31 Oct 2012 at 10:07

**Comment by GoogleCodeExporter** [comment]
```
I just implemented the Zine 2.0.3 version but it still display small squares at 
some space characters.

Here's a screen shot:
http://cl.ly/image/3D2q251L441r


```

Original comment by `maartenw...@gmail.com` on 6 Nov 2012 at 12:48
